addons/account/models/account_move.py

```diff
@@ -3130,17 +3178,53 @@ def _compute_has_reconciled_entries(self):
     def _autopost_draft_entries(self):
         ''' This method is called from a cron job.
         It is used to post entries such as those created by the module
-        account_asset.
+        account_asset and recurring entries created by this very method.
```

Comment:
Not "by this very method" anymore
Unless you refer to `post`?

Reply:
I should write the docs once everything else is done.